code/modules/paperwork/paper_reader.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/paper_reader
+	name = "eletronic paper reader"
+	gender = NEUTER
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "paper_reader"
+	throwforce = 0
+	w_class = WEIGHT_CLASS_TINY
+	pressure_resistance = 0
+	max_integrity = 50
+	///TODO list for the reading loop
+	var/list/to_read = list()
+	///Cooldown for use
+	var/read_cooldown = 4 SECONDS
+	var/cooldown_timer
```

Comment:
redundant

Also, I'd like to suggest 2 new vars to replace this behaviour.
```js
/// When an item is used, this will be flagged to be TRUE, and becomes FALSE when it no longer runs.
var/on_running
/// When on_running is TRUE, using the item again will turn on this. This will stop the item running.
var/try_interrupt
```